src/main/drivers/pwm_output.h

```diff
@@ -179,7 +180,7 @@ void pwmCompleteDshotMotorUpdate(uint8_t motorCount);
 #ifdef BEEPER
 void pwmWriteBeeper(bool onoffBeep);
 void pwmToggleBeeper(void);
-void beeperPwmInit(IO_t io, uint16_t frequency);
+void beeperPwmInit(const beeperDevConfig_t *beeperConfig);
```

Comment:
`pwm_output` should not have a dependency on beeper. Please change `beeperPwmInit` back to `beeperPwmInit(IO_t io, uint16_t frequency);` and dereference `beeperDevConfig_t` in `sound_beeper.c`